gnd/src/main/java/com/google/android/gnd/workers/TileDownloadWorker.java

```diff
@@ -155,6 +156,13 @@ private Completable downloadIfNotFound(Tile tile) {
 
   private Completable processTiles(ImmutableList<Tile> pendingTiles) {
     return Observable.fromIterable(pendingTiles)
+        .doOnSubscribe(__ -> sendNotification(UploadProgress.starting()))
```

Comment:
A little weird that this is called UploadProgress from this context, since we're actually downloading tiles, not uploading them. Of course, for photos we're performing an upload not a download...

Maybe make the name more generic?

Reply:
TransferProgress?

Reply:
Thanks!

Reply:
1b72b05